packages/p2p-v2/src/pages/advertiser/screens/Advertiser/Advertiser.scss

```diff
@@ -1,6 +1,13 @@
 .p2p-v2-advertiser {
+    position: absolute;
+    top: 8rem;
+    background-color: #fff;
+    width: 95.2rem;
+
     @include mobile {
-        height: calc(100vh - 8rem);
+        top: 4rem;
         overflow-y: scroll;
+        height: calc(100vh - 8rem);
+        width: 100%;
```

Comment:
why are we changing these? and is this related to the Router component?

Reply:
yeah it got affected because of how I structured the way Routers sit with Tabs. Tabs are sat on top and no longer conditionally render the Tab components, while Router is at the bottom which renders all the pages. And because of that, Advertiser is placed under the tabs, so I had to position it to hover over the tabs